lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -64,6 +64,20 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
             // NodeOp to add the InnerSym.
             if (!xmrDef.isa<BlockArgument>()) {
               Operation *xmrDefOp = xmrDef.getDefiningOp();
+              if (auto verbExpr = dyn_cast<VerbatimExprOp>(xmrDefOp))
+                if (verbExpr.getSymbolsAttr().empty() &&
+                    xmrDefOp->hasOneUse()) {
+                  // This represents the internal path into a module. For
+                  // generating the correct XMR, no node can be created in this
+                  // module. Create a null InnerRef and ensure the hierarchical
+                  // path ends at the parent that instantiates this module.
+                  auto inRef = InnerRefAttr();
+                  auto ind = addReachingSendsEntry(send.getResult(), inRef);
+                  xmrPathSuffix[ind] = ("." + verbExpr.getText()).str();
```

Comment:
Is there a situation where the PR shouldn't add a `"."` leading prefix?

Reply:
You are right, we always need it. So, it can potentially be moved to the use of the suffix.